codejam-canvas/scripts/script.js

```diff
@@ -0,0 +1,51 @@
+document.addEventListener('DOMContentLoaded', event => {
+    let canvasContext = getCanvasContext();
+    let radioButtons = document.getElementsByTagName('input');
+    radioButtons[0].onclick = () => fetch("assets/data/4x4.json")
+        .then(response => response.json().then(json => drawMatrix4x4(json, canvasContext)));
+    radioButtons[1].onclick = () => fetch("assets/data/32x32.json")
+        .then(response => response.json().then(json => drawMatrix32x32(json, canvasContext)));
+    radioButtons[2].onclick = () => drawCanvasImage(canvasContext);
```

Comment:
1) use addEventListener, onclick fufufu
2) then inside then cannot be used, u should use chaining then -> ().then().then()